docs/the_black_code_style/current_style.md

```diff
@@ -200,6 +200,15 @@ following field or method. This conforms to
 _Black_ won't insert empty lines after function docstrings unless that empty line is
 required due to an inner function starting immediately after.
 
+### Comments
+
+_Black_ does not format comment contents, but it enforces two spaces between code and a
+comment on the same line, and a space after the comment text begins. Some comments that
+are known to require contradictory spacing rules, like doc comments, are respected.
```

Comment:
Should we provide an exhaustive (or at least more comprehensive) list, or just briefly mention special comments? From what I could gather there are at least section comments that are not modified, i.e. long strings of `#####` or `### text ###`, non-breaking spaces and Spyder cells (don't actually have a clue about those),

Reply:
"Some comments that require specific spacing rules --" or "Some types of comments --" could also be better.

Reply:
I think listing them all is going to be helpful. There are several open issues asking for more similar processing rules (under the "comments" label in the issue tracker), so people will probably want to know what we already support.

Reply:
There you go! Let me know if I've missed anything. Now that I think about it, it could be best to refactor at least the tests for special comments into one place. But I think that's for another PR.